debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/sink/event_handler/EventHandler.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb.sink.event_handler;
```

Comment:
But it's valid and not in the styleguide. So I think I can leave it? ;)

Reply:
It's definitively unusual and I too think it looks weird. I'm not objecting per se. However, I'd suggest looking at whether we already have a "multi-word" package names already in the code base and use the same style in order to be consistent

Reply:
Well, thanks for the reminder. We are going to add this into the styleguide as something that should not be used :-)

Reply:
Isn't this kind of bias based comment? I would personally find `record_emitter` much more readable than `recordemitter` especially when you are new to the code, not a native speaker or not aware of the specific context.
The official Java docs from Oracle even recommend underscores in package names when those originate from real domain names that have dashes in the name:
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

I think only Google/Google's Java styleguide prefers concatenation.

Reply:
Perhaps it may make more sense to find a better package name to avoid the need to even discuss this point.  We have packages like `converters` and `types`, which are super generic; why not just use `handlers` or some other meaning _word_ and avoid the use of two words altogether?

Reply:
You are right, it is. So we need to reach a quorum -- or have Jiri overrule as democracy gets you only so far :).

Reply:
@rk3rn3r Thanks